subsys/bluetooth/host/Kconfig.gatt

```diff
@@ -43,6 +43,14 @@ config BT_EATT_MAX
 	help
 	  Number of Enhanced ATT bearers available.
 
+
+config BT_EATT_AUTO_CONNECT
+	bool "Automatically connect EATT bearers when a link is established"
+	default y
+	help
+	  The central will try to connect BT_EATT_MAX enhanced ATT bearers when a
+	  connection to a peripheral is established.
```

Comment:
So this is only for the central? 

If so, then we should add dependency on `BT_CENTRAL`. 
Is there any reason why the peripheral shouldn't attempt to create the EATT bearers/channels?

Reply:
Chose that only the central initiates the request to avoid collisions. With two zephyr devices this is handled by the L2CAP collision mitigation procedure, but it may not work as well with other devices.

Bluez also chose to only auto-connect if it is the central.

Reply:
The alternative is of course to have a delay for one of the sides (e.g. 2 x connection interval delay on the peripheral). 

IIRC there are no recommendations that the central or peripheral should attempt to create these on connection. In the case of LE Audio where we definitely want to use EATT channels, it may be useful for the peripheral side to also auto-initiate these in case we are connected to a central that doesn't connect them.

Reply:
Yes, we can let the peripheral delay the connection request as is done in the collision mitigation procedure. This will work fine with two zephyr devices. Depending on what other stacks does we will have to rely on the collision mitigation working.

Reply:
> Depending on what other stacks does we will have to rely on the collision mitigation working.

We will need to do that regardless in case that we connect to a non-Zephyr peripheral that auto-connects without delay as well :)

Reply:
Changed it so both central and peripheral tries to connect. We may want to consider a backoff-mechanism in the future if the spec-required collision mitigation is insufficient.